pennylane/plugins/default_gaussian.py

```diff
@@ -836,9 +836,43 @@ def var(self, observable, wires, par):
         _, var = self._observable_map[observable](mu, cov, wires, par, hbar=self.hbar, total_wires=self.num_wires)
         return var
 
+
     def sample(self, observable, wires, par, n=None):
-        raise NotImplementedError("Sampling is not supported in default.gaussian, "
-                                  "please install PennyLane-SF or another plugin capable of sampling")
+        r"""Returns n samples of the given observable in wires.
```

Comment:
Note: in general, we prefer not to include docstrings of inherited methods. Python and Sphinx will automatically 'inherit' the docstring if not provided.

The one exception is if the overwritten method changes the functionality or the signature in a way that should be communicated to the user. In this case, it might be worth keeping the docstring, but adding a `.. note::` to explain that sampling is only supported for `qml.X`, `qml.P`, and `qml.QuadOperator`.

Reply:
```suggestion
        r"""Returns n samples of the given observable in wires.
        
        .. note::
        
            The ``default.gaussian`` plugin currently only supports sampling
            from the `qml.X`, `qml.P`, and `qml.QuadOperator` observables.
```